cmd/rollapp/init/utils.go

```diff
@@ -68,61 +66,6 @@ func runInit(
 
 	addresses = append(addresses, sequencerKeys...)
 
-	/* ------------------------ Initialize DA light node ------------------------ */
```

Comment:
why the move to setup?

Reply:
because the NewDAManager function now needs to be set after we set nodetype to initialize the corresponding DA for the node @artemijspavlovs